docs/src/main/asciidoc/spanner.adoc

```diff
@@ -1038,6 +1038,8 @@ The following filter options are supported:
 * Not like a string
 * Contains a string
 * Not contains a string
+* In
+* Not in
```

Comment:
we should document `IN UNNEST` syntax and when to use it instead of `IN` for @Query annotated methods

Reply:
done.